src/Console/stubs/directive.stub

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace DummyNamespace;
+
+use Nuwave\Lighthouse\Schema\Directives\BaseDirective;
+
+class DummyClass
```

Comment:
This should extend `BaseDirective`, which makes implementation much simpler.

Reply:
Whoops, forgot about that. Thank you!